setup.py

```diff
@@ -25,6 +25,6 @@
       license='3-Clause BSD',
       packages=['splot'],
       include_package_data=True,
-      install_requires=['numpy', 'libpysal', 'mapclassify', 'palettable',
```

Comment:
to make sure, after many of these rebase fixes, you know you're dropping palettable and adding spreg as dependencies here?

Reply:
that happens in pr #23  ;)

Reply:
OK!